content/applications/productivity/iot/config/connect.rst

```diff
@@ -1,81 +1,145 @@
-===================================
-Connect an IoT Box to your database
-===================================
+============================================================
+Connect an Internet of Things (IoT) box to the Odoo database
+============================================================
 
-Install the Internet of Things (IoT) App on your Odoo Database.
+The Internet of Things (IoT) box is a micro-computer device that allows for the connection of
+devices to a Odoo database. A subscription is required in order to use the
+:abbr:`IoT (Internet of Things)` box.
+
+Install the :menuselection:`Internet of Things (IoT)` App on the Odoo database.
 
 .. image:: connect/connect01.png
    :align: center
+   :alt: The Internet of Things (IoT) App on the Odoo database.
 
-Go in the IoT App and click on Connect on the IoT Boxes page.
+Go to the :menuselection:`IoT` App and click on :guilabel:`Connect` on the
+:abbr:`IoT (Internet of Things)` IoT Boxes page.
 
-.. image:: connect/connect02.png
+.. image:: connect/connect09.png
    :align: center
+   :alt: Connecting an IoT box to the Odoo database.
 
-Follow the steps to connect your IoT Box.
+Follow the steps to connect the IoT Box either via wired ethernet connection or via WiFi.
 
-.. image:: connect/connect03.png
+.. image:: connect/connect-iot-box.png
    :align: center
+   :alt: Connection steps for a wired connection or WiFi connection.
 
-Ethernet Connection
-===================
+.. note::
+   Ensure the :abbr:`IoT (Internet of Things)` box is flashed with the most up-to-date
+   :abbr:`IoT (Internet of Things)` image.
```

Comment:
for a non tech people, maybe the term "image" can be confusing ? Maybe we should use a more generic term like "version" ?